### DISCUSSION TITLE
When reviewing events, the screen moves up

### DISCUSSION THREAD

**Discussion by FS1961** [blakeblackshear/frigate#13719] [Ask A Question] [answered]
Please see the photo. When I access the UI from HA, I can see the events fine. If I select one, the screen slips higher? 
![IMG_0542](https://github.com/user-attachments/assets/215de022-b4a0-4abc-b74b-da55ee62b5dd)

System 0.14.1-f4f3cfa
Apple companion app  5.2024

not sure if this is a Frigate, HA add-in, or companion app issue.

**Comment by NickM-27** [comment, marked as answer]
duplicate of https://github.com/blakeblackshear/frigate/discussions/13199
